### ISSUE TITLE
Add to linkml-runtime

### ISSUE THREAD

**Issue by hsolbrig** [linkml/linkml-csv#1]
@cmungall  -- I'm going to work with Joe next week and, together, we'll propose a solution to get the linkml dependency out of this.   In a nutshell, we're going to use the yaml_dumper to emit a fully parsed image of the schema you are using and will import it with the runtime yaml_loader for what is going on here.

Just a training proposal but hopefully it will be a pattern we can use going forward

**Comment by cmungall** [comment]
Thanks! 

The reason I didn't go with the approach above is that the raw view doesn't make use of things like inheritance hierarchies

However, if we can merge
https://github.com/linkml/linkml-runtime/pull/25

Then we can use the schemaview induced_slot method to get the full inferred properties for each slot

**Comment by cmungall** [comment]
I just sent @joeflack4 an invite to the link developer team on github

**Comment by joeflack4** [comment]
@cmungall @hsolbrig Hey guys. I just got notified of the assignment, thanks. I've reached out to Harold on Skype but haven't heard back yet. Harold, I know that this must be a busy and probably stressful week for you!

I'll stay posted for more information. But perhaps what is being asked of me is as simple as moving the `linkml_csv` folder/module into the `linkml_runtime` package's directory. And then I would move the tests where they need to go. And perhaps I would need to change a few other small things, such as Pipenv, __init__ imports. Not sure but I'd have to take a deeper look / know more information about what's required.
